modules/nw-ovn-ipsec-enable.adoc

```diff
@@ -23,3 +23,41 @@ As a cluster administrator, you can enable IPsec encryption after cluster instal
 $ oc patch networks.operator.openshift.io cluster --type=merge \
 -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"ipsecConfig":{ }}}}}'
 ----
+
+.Verification
+
+. To find the names of the OVN-Kubernetes control plane pods, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods -l app=ovnkube-master -n openshift-ovn-kubernetes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                   READY   STATUS    RESTARTS   AGE
+ovnkube-master-fvtnh   6/6     Running   0          122m
+ovnkube-master-hsgmm   6/6     Running   0          122m
+ovnkube-master-qcmdc   6/6     Running   0          122m
+----
+
+. Verify that IPsec is enabled on your cluster:
```

Comment:
You've got ", enter the following command:" in the previous step. I'd go with that (or switch both to "by running the following command:") here as well.

Reply:
I was just moving this content, but I agree for consistency.